DeploymentCloud/Deployment.Common/CosmosDB/commons.json

```diff
@@ -701,5 +717,171 @@
+					},
+					"process" : {
+						"metric" : {
+							"eventhub" : 
+							"keyvault://${sparkKeyVaultName}/${metricEventHubConnectionStringKey}"
+						},
+						"timestampColumn" : "${processTimestampColumn}",
+						"watermark" : "${processWatermark}",
+						"jarUDAFs" : "${processJarUDAFs}",
+						"jarUDFs" : "${processJarUDFs}",
+						"azureFunctions" : "${processAzureFunctions}",
+						"projections" : "${processProjections}",
+						"timeWindows" : "${processTimeWindows}",
+						"transform" : "${processTransforms}",
+						"appendEventTags" : {},
+						"accumulationTables" : "${processStateTables}"
+					},
+					"outputs" : "${outputs}"
+				},
+				"sparkJobTemplateRef" : "DataXBatch",
+				"jobCommonTokens" : {
+					"jobName" : "${name}",
+					"sparkJobName" : "DataXBatch-${name}",
+					"sparkJobDriverLogLevel" : "WARN",
+					"sparkJobExecutorLogLevel" : "INFO",
+					"sparkJobNumExecutors" : "${guiSparkJobNumExecutors}",
+					"sparkJobExecutorMemory" : "${guiSparkJobExecutorMemory}",
+					"processedSchemaPath" : null
+				},
+				"jobs" : [ 
+					{
+						"partitionJobNumber" : "1"
+					}
+				]
+			},
+			"metrics" : {
+				"sources" : [ 
+					{
+						"name" : "events",
+						"input" : {
+							"type" : "MetricApi",
+							"metricKeys" : [ 
+								"DATAX-${name}:Input_DataXProcessedInput_Events_Count"
+							]
+						},
+						"output" : {
+							"type" : "SumWithTimeChart",
+							"data" : {
+								"sum" : true,
+								"timechart" : true,
+								"average" : true,
+								"speed" : true
+							}
+						}
+					}
+				],
+				"widgets" : [ 
+					{
+						"name" : "eventsChart",
+						"displayName" : "Events/Second",
+						"data" : "events_timechart",
+						"position" : "TimeCharts",
+						"type" : "StackAreaChart"
+					}, 
+					{
+						"name" : "totalEvents",
+						"displayName" : "Events Ingested Today",
+						"data" : "events_sum",
+						"formatter" : "longint",
+						"type" : "SimpleBox",
+						"position" : "FirstRow"
+					}, 
+					{
+						"name" : "averageEvents",
+						"displayName" : "Avg. Events/Minute",
+						"data" : "events_average",
+						"formatter" : "longint",
+						"type" : "SimpleBox",
+						"position" : "FirstRow"
+					}
+				],
+				"initParameters" : {
+					"widgetSets" : [ 
+						"direct"
+					],
+					"jobNames" : {
+						"type" : "getCPSparkJobNames"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name" : "DataXBatch",
+		"content" : {
+			"name" : "${sparkJobName}",
+			  "cluster": "$sparkName",
+			"options" : {
+				"name" : "${sparkJobName}",
+				"file" : "wasbs:///datax/bin/datax-host_2.3_2.11-1.1.0-SNAPSHOT.jar",
+				"className" : "datax.app.BatchApp",
+				"args" : [ 
+					"conf=${sparkJobConfigFilePath}", 
+					"driverLogLevel=${sparkJobDriverLogLevel}"
+				],
+				"jars" : [ 
+					"wasbs:///datax/bin/datax-core_2.3_2.11-1.1.0-SNAPSHOT.jar", 
+					"wasbs:///datax/bin/datax-utility_2.3_2.11-1.1.0-SNAPSHOT.jar", 
```

Comment:
We'll need to remove the snapshots when going to stable.  Can you take a note

Reply:
Sure, yes it will be a part of the release process.

Reply:
now I remembered that even for Master, we shouldn't have the snapshots. will update them.

Reply:
@rohit489 can you confirm?

Reply:
ARM template should not include snapshots as even for master branch we fetch packages from our public nuget feed which